### ISSUE TITLE
Port maemo-launcher init script to OpenRC

### ISSUE THREAD

**Issue by MerlijnWajer** [maemo-leste/bugtracker#201]
Make sure to check that:
* reload also still works
* figure out postinst should always start/reload on install, as this is currently causing issues in jenkins

**Comment by freemangordon** [comment]
I don't think maemo-launcher should be installed on build env, so start on package install/restart on upgrade is fine IMO

**Comment by MerlijnWajer** [comment]
agreed, but we'll still have to make sure the script is ported to openrc. :)
